docs/app/pages/getting-started/getting-started.component.html

```diff
@@ -1,84 +1,115 @@
+                <li>Install the UX Aspects package and its dependencies: <kbd>npm install @ux-aspects/ux-aspects --save</kbd></li>
+            </ol>
+        </tab>
+        <tab heading="Bower">
+            <ol>
+                <li>Install <a href="https://nodejs.org/">Node.js</a>, which provides the npm package manager.</li>
+                <li>Install <a href="https://www.typescriptlang.org/">TypeScript</a>: <kbd>npm install typescript --save-dev</kbd></li>
+                <li>Install <a href="https://angular.io/">Angular</a> and other dependencies:
+                    <div>
+                        <kbd>npm install @angular/common @angular/compiler @angular/compiler-cli @angular/core @angular/forms
+                            @angular/http @angular/platform-browser @angular/platform-browser-dynamic @angular/platform-server
+                            @angular/router chart.js ngx-bootstrap ng2-charts @types/chart.js --save</kbd>
+                    </div>
+                </li>
+                <li>Install <a href="https://bower.io/">Bower</a>: <kbd>npm install -g bower</kbd></li>
+                <li>Install the UX Aspects bower package: <kbd>bower install ux-aspects#{{ version }}</kbd></li>
+            </ol>
+        </tab>
+        <tab heading="Direct Download">
+            <p>The UX Aspects resources (scripts, CSS, fonts, and images) can be downloaded from the
+                <a href="https://github.com/UXAspects/UXAspects/releases">GitHub releases page</a>.</p>
+        </tab>
+        <tab heading="Building from Source">
+            <ol>
+                <li>Clone the <a href="https://github.com/UXAspects/UXAspects">GitHub repository</a>.</li>
+                <li>Install the dependencies: <kbd>npm install</kbd></li>
+                <li>Build the project: <kbd>grunt build</kbd>. The resources (scripts, CSS, fonts, and images) can be found in the
+                    <code>dist</code> folder.
+                </li>
+            </ol>
+        </tab>
+    </tabset>
+
+    <h3>Applying the Stylesheet</h3>
+
+    <div class="row-divider"></div>
+
+    <p>The UX Aspects stylesheet can be applied in four different ways as outlined in the following tabs:</p>
+
+    <tabset>
+        <tab heading="Angular CLI">
+            <p>If using the CLI, the UX Aspects stylesheet can be added in the <code>angular-cli.json</code> file:</p>     
+            <uxd-snippet [code]="cliJson" language="javascript"></uxd-snippet>
+        </tab>
+        <tab heading="Webpack">
+            <p>When using Webpack the UX Aspects stylesheet can be included in the main stylesheet of the application:</p>          
+            <uxd-snippet [code]="stylesheetCss" language="css"></uxd-snippet>
+        </tab>
+        <tab heading="Non-Angular Sites">
+            <p>If your application does not use Node or Webpack, you can copy the UX Aspects stylesheet into your site's assets 
```

Comment:
We can actually probably change this to adding the stylesheet from the bower components folder using a link tag in the page header. 

Unlike node_modules, bower_components folder is usually deployed with the application so you can just point directly to the stylesheet in the bower_components/uxaspects folder. Copying the stylesheet to an assets folder would not work unless you also copied the fonts folder and img folder etc..